tests/src/utility/auth-utils.ts

```
@@ -54,6 +54,18 @@ export async function findPageWithTitleInBrowser(browser: Browser, expectedTitle
     await input.fill(username);
     const nextButton = page.getByRole('button', { name: 'Next' });
     await nextButton.click();
+    // after next is clicked, a wait is necessary
+    // we might get a page with providers to choose from
+    // button - Log in with company single sign-on OR
+    // button - Log in with Red Hat account
+    const buttonRHAccount = page.getByRole('button', { name: 'Log in with Red Hat account' });
+    // we might get to the password immediately 
+    try {
+      await playExpect(buttonRHAccount).toBeVisible({ timeout: 5000 });
+      await buttonRHAccount.click();
+    } catch (error: unknown) {
+      console.log(`Error: ${error}, while evaluating Log in with RH acc. button, continue...`);
```

We are swallowing the error is that one purpose ?